### PR TITLE
Custom checkpoints

### PR DESCRIPTION
**Pull request by stevensJourney**
# Overview

This adds a demonstration for using the , Enterprise,  Custom Write Checkpoints feature.

This requires the changes from https://github.com/powersync-ja/powersync-nodejs-backend-todolist-demo/pull/10

### CODE REVIEW

File: demos/nodejs-custom-checkpoints/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# JavaScript PowerSync + Custom Write Checkpoints
+
+This is a demo for using custom Write Checkpoints with PowerSync. Custom Write Checkpoints eliminate flicker when uploads do not directly reflect in the backend source database. This feature is only available on Enterprise plans.
```
[the remark this answers is not in the document]
Comment (benitav) [reply demos/nodejs-custom-checkpoints/README.md:3]: wherever we say Enterprise we also need to include the Team plan

[the remark this answers is not in the document]
Comment (stevensJourney) [reply demos/nodejs-custom-checkpoints/README.md]: Thanks, I've updated all references to the above suggestion.

File: demos/nodejs-custom-checkpoints/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# JavaScript PowerSync + Custom Write Checkpoints
+
+This is a demo for using custom Write Checkpoints with PowerSync. Custom Write Checkpoints eliminate flicker when uploads do not directly reflect in the backend source database. This feature is only available for customers on [Team and Enterprise](https://www.powersync.com/pricing) plans.
```
Comment (benitav) [inline demos/nodejs-custom-checkpoints/README.md:3]:
```suggestion
This is a demo for using custom Write Checkpoints with PowerSync. Custom Write Checkpoints eliminate flicker when uploads do not directly reflect in the backend source database. This feature is only available for customers on [Team and Enterprise](https://www.powersync.com/pricing) plans.

Docs are available [here](https://docs.powersync.com/usage/use-case-examples/custom-write-checkpoints).
```

Comment (benitav) [reply demos/nodejs-custom-checkpoints/README.md:3]: Not yet available, but that's where they should be after https://github.com/powersync-ja/mintlify-docs/pull/11/files

**Review by benitav — DISMISSED** [review]

**Review by rkistner — DISMISSED** [review]

**Review by benitav — APPROVED** [review]

**Pull request merged**